src/modules/PowerOCR/PowerOCRModuleInterface/Resources.resx

```diff
@@ -117,10 +117,10 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="PowerOCR_Name" xml:space="preserve">
-    <value>PowerOCR</value>
+  <data name="ImageToText_Name" xml:space="preserve">
+    <value>Image To Text</value>
```

Comment:
Please add comment that we mean to like convert to. Otherwise we might get translation problems because to can have different meanings (send to, convert to, copy to).